Documentation/subcommands/run.md

```diff
@@ -129,7 +129,22 @@ If a mount point is specified in the image manifest but no matching volume is fo
 
 ### Mounting Volumes
 
-For `host` volumes, the `--volume` flag allows you to specify each mount, its type, the location on the host, and whether the volume is read-only or not.
+Volumes are defined via the `--volume` flag, the volume is then mounted into each app running to the pod based on information defined in the ACI manifest.
```

Comment:
I don't quite understand the "running to the pod" part. Should it be "running in the pod"?